environment/src/test/java/com/sequenceiq/environment/service/integration/CredentialAuthorizationIntegrationTest.java

```diff
@@ -122,8 +122,8 @@ public void testCredentialPermissions() {
         credentialRepository.save(getAwsCredential(FIRST_CRED_NAME, ACCOUNT_ID, FIRST_USER_CRN));
         credentialRepository.save(getAwsCredential(SECOND_CRED_NAME, ACCOUNT_ID, SECOND_USER_CRN));
 
-        testUnhappyPaths(firstUserClient, SECOND_CRED_NAME);
-        testUnhappyPaths(secondUserClient, FIRST_CRED_NAME);
+//        testUnhappyPaths(firstUserClient, SECOND_CRED_NAME);
+//        testUnhappyPaths(secondUserClient, FIRST_CRED_NAME);
```

Comment:
What was the issue with these? Credential should work like before I think

Reply:
It was a mistake, I forgot to uncomment these two lines! Thanks Adam!